lib/tests/conftest.py

```diff
@@ -87,8 +86,6 @@ def pytest_configure(config: pytest.Config):
 
     is_require_snowflake = config.getoption("--require-snowflake", default=False)
     if is_require_snowflake:
-        if sys.version_info[0:2] != (3, 8):
```

Comment:
This was a limitation with an old snowpark version which was only working with Python 3.8. The most recent one works across various Python versions.